src/selectors/shelf.test.ts

```diff
@@ -16,6 +17,7 @@ describe('selectors/shelf', () => {
         undoable: {
           ...DEFAULT_STATE.undoable,
           present: {
+            customWildcardFields: DEFAULT_CUSTOM_WILDCARD_FIELDS,
```

Comment:
Instead of adding this, add `...DEFAULT_UNDOABLE_STATE_BASE` and remove `result: DEFAULT_RESULT_INDEX`